datafusion/functions/benches/to_char.rs

```diff
@@ -17,20 +17,16 @@
 
 extern crate criterion;
 
-use std::sync::Arc;
-
-use arrow_array::{ArrayRef, Date32Array, StringArray};
 use chrono::prelude::*;
 use chrono::TimeDelta;
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
 use rand::rngs::ThreadRng;
 use rand::seq::SliceRandom;
 use rand::Rng;
 
-use datafusion_common::ScalarValue;
 use datafusion_common::ScalarValue::TimestampNanosecond;
-use datafusion_expr::ColumnarValue;
-use datafusion_physical_expr::datetime_expressions::to_char;
+use datafusion_expr::{lit, Expr};
+use datafusion_functions::expr_fn::to_char;
```

Comment:
I think this is the wrong function to be benchmarking -- it should be the implementation (not the expr_fn)